tests/data/usavars/elevation.csv

```diff
@@ -0,0 +1,3 @@
+,Unnamed: 0,ID,lon,lat,elevation
```

Comment:
Can you take out this "Unnamed: 0" column in the test datasets (unless that is also part of the actual dataset CSV format)